LoRaEngine/modules/LoRaBasicsStationModule/start_basicsstation.sh

```diff
@@ -28,15 +34,16 @@ fi
 
 #start basestation
```

Comment:
i know it's not your code 
```suggestion
#start basics station
```